sample/src/components/emulator.tsx

```diff
@@ -1,23 +1,25 @@
-import React, { FunctionComponent } from 'react'
-import ReactGbaJs from 'react-gbajs'
+import React, { FunctionComponent } from 'react';
+import ReactGbaJs from 'react-gbajs';
 
-type Props = { scale: number, printFps: boolean }
+type Props = { scale: number; printFps: boolean };
 const Emulator: FunctionComponent<Props> = ({ scale, printFps }) => {
-  const fpsReportCallback = (
-    printFps
-      ? (fps: number) => console.log('FPS', fps)
-      : undefined
-  )
+  const fpsReportCallback = printFps
+    ? (fps: number) => console.log('FPS', fps)
+    : undefined;
 
   return (
     <ReactGbaJs
       onFpsReported={fpsReportCallback}
       scale={scale}
-      volume={0}
+      volume={1}
```

Comment:
I would prefer to keep the example with the default values.
In this case, the default `volume` is `0`, and then here should be kept as `1`